src/parser/jobs/ast/modules/Draw.tsx

```diff
@@ -15,7 +15,7 @@ import ArcanaTracking from './ArcanaTracking/ArcanaTracking'
 // Track them using Draw when they still have a minor arcana (oopsie) or a card in the spread
 
 const CARD_DURATION = 15000
-const SLEEVE_DRAW_PLAYS_GIVEN = 3
+const SLEEVE_DRAW_PLAYS_GIVEN = 1
```

Comment:
Is this patch conditional? You can take a look at MNK's Forms module and use a ternary or something to switch it out as appropriate.

Reply:
You're right, this will break prepatch. I'll check out punchypunch